### ISSUE TITLE
Feature: Compute & export data for bivariate plots

### ISSUE THREAD

**Issue by vejnar** [wdecoster/cramino#22]
Thanks for developing cramino. It's very fast! The bivariate plots generated by Nanoplot are very useful but the arrow/feather file computed by cramino doesn't have that info. Could it possible to compute, or add, if already available in cramino, the bivariate data to the arrow/feather file? Nanoplot could then be used for plotting only.

**Comment by wdecoster** [comment]
Which plots, specifically? The arrow file should contain the information to show the length vs accuracy plot. Was that not generated?

**Comment by vejnar** [comment]
Thanks for your reply. Using the options `--arrow` and `plots kde`, I am getting 8 plots, but in addition when running Nanoplot directly on the same BAM file, I got these plots:
* Percent identity x Average base quality
* Read length x Read mapping quality
* Read mapping quality x Average basecall quality
* Aligned read lengths x Sequenced read length
* Read lengths x Average read quality
And at the top of the report, I also got stats of reads above quality cutoffs and different top 5.

Maybe I am missing something, but I am not sure what option of cramino or Nanoplot I could change. I am using the same options for Nanoplot, excluding the `--bam` vs `--arrow` options.

Thanks for your help.
